src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,25 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Comments } from '../../types/Comments';
+
+type Props = {
+  comments: Comments[];
+  id: number;
+};
+
+export const CommentList: React.FC<Props> = ({ comments, id }) => {
+  const userComents = comments.filter(comment => comment.postId === id);
+
+  return (
+    <div className="CommentList">
+      {
+        userComents.length > 0
+          ? userComents.map(comment => (
+            <CommentInfo {...comment} key={comment.id} />
+          ))
+          : 'No Comments Message'
```

Comment:
Change the 'No Comments Message' to a more user-friendly message, such as 'No comments available for this post.'